core/src/com/unciv/logic/map/mapgenerator/MapRegions.kt

```diff
@@ -1452,7 +1452,7 @@ class MapRegions (val ruleset: Ruleset){
                 // If there is no matching list, it is because the rule was determined to be impossible and so can be safely skipped
                     ?: continue
                 // Place the resources
-                placeResourcesInTiles((rule.params[0].toFloat() * bonusMultiplier).toInt(), list, listOf(resource), 0 + extraImpact, 2 + extraImpact, false)
+                placeResourcesInTiles(maxOf((rule.params[0].toFloat() * bonusMultiplier).toInt(),1), list, listOf(resource), 0 + extraImpact, 2 + extraImpact, false)
```

Comment:
there is no division here so I also can't see why this would give that error

Reply:
`placeResourcesInTiles(frequency: Int, ...)` has a line `val amountToPlace = (tileList.size / frequency) + 1`, although it might be better for him to put the fix in the function itself since the function is called in a few places.